### ISSUE TITLE
Zooming with finger over toggle switch moves screen

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecule-polarity#132]
**Device**
iPad 6th Gen
**OS**
iPadOS 14
**Browser**
Safari
**Problem Description**
For https://github.com/phetsims/qa/issues/688. When zooming in or out with pinch to zoom, if your finger starts on the toggle switch for the electric field, the zoomed in screen's position will jump down to the bottom right seemingly at random.
**Visuals**

https://user-images.githubusercontent.com/41024075/129389591-677a6733-ee56-466d-aeaf-e65bf6c82995.MOV

**Comment by pixelzoom** [comment]
I have no idea why this would occur only with the ABSwitch.  And I'm not at all familiar with zoom support.  So we'll need to involve @jessegreenberg.  High priority, since this is for an RC-test cycle.

**Comment by pixelzoom** [comment]
@arouinfar please comment on whether this blocks sim publication -- i.e., does it need to be fix, and require verification in another RC.

**Comment by pixelzoom** [comment]
This sounds like a general problem, since there's nothing specific that this sim is doing for ABSwitch or zoom. 

@KatieWoe Could you please test whether this occurs with other sims that use ABSwitch?  Some examples are: pH Scale, acid-base-solutions, area-builder, balancing-act, color-vision.  Testing in master would be fine.

**Comment by arouinfar** [comment]
It looks like the pan and zoom feature is not working as intended, but not in a way that is sim-breaking. I am fine with publishing Molecule Polarity with this issue. 

I cannot reproduce this behavior on my MacBook using the pinch-to-zoom gesture on my trackpad. I would like @KatieWoe to investigate more before sending this off to @jessegreenberg.

@KatieWoe 
* Is this reproducible on other devices?
* If so, does it only happen on touch devices?
* Can you get it to happen on other ABSwitches? Please test the sims @pixelzoom suggested on master. Additionally, test the latest published version of pH Scale.

**Comment by KatieWoe** [comment]
- I was able to reproduce this on other sims that were mentioned by @pixelzoom in master
- I was able to reproduce on the touchscreen of Win 10 with Chrome Browser
- I'm not certain how it would be reproduced without touch, but I wasn't able to see it with a trackpad on the same Win 10 device
- I tried a few other elements and I saw similar behavior when my finger was over a slider in acid-base-solutions

Sorry this wasn't seen earlier

**Comment by pixelzoom** [comment]
The interactive part of ABSwitch is ToggleSwitch.  I don't see anything unusual on ToggleSwitch, or anything specific to zoom.   So I'm wondering if there is a general problem with trying to zoom while you're dragging something.  

@KatieWoe Could you please try to reproduce this while dragging some things? A slider, a molecule,... basically anything that is draggable.   Also please test panning, in case there's a problem there too.

**Comment by KatieWoe** [comment]
I think you're right. I managed to reproduce this dragging the molecule in molecule polarity and a circuit component in cck:dc (both out of the toolbox and in the play area). It seems to happen more if the first finger that comes down is the one over the element being dragged.

**Comment by pixelzoom** [comment]
Since @arouinfar identified that this is not blocking for Molecule Polarity 1.2, I'm going to proceed and publish with this known problem. 

And since this is clearly a general problem, the next step is to migrate this to the appropriate common-code repo -- probably scenery.

**Comment by pixelzoom** [comment]
Migrating this issue from molecule-polarity to scenery.
